features/validate_case.feature

```diff
@@ -0,0 +1,78 @@
+Feature: Validate case with URN, offences, postcode and date of birth
+
+  Background:
+    Given a browser
```

Comment:
Can we combine these two steps?
Given a browser
When I visit "plea/enter_urn/"